Source/Core/Core/BootManager.cpp

```diff
@@ -300,6 +305,9 @@ bool BootCore(const std::string& _rFilename)
         g_wiimote_sources[WIIMOTE_BALANCE_BOARD] = source;
         WiimoteReal::ChangeWiimoteSource(WIIMOTE_BALANCE_BOARD, source);
       }
+
+      IniFile::Section* sysconf_section = game_ini.GetOrCreateSection("Sysconf");
```

Comment:
Do we already have a section called "Sysconf", or is that new for this PR? If we don't already have one, would to be more appropriate to use an existing section?